### ISSUE TITLE
[Feature request]: Larger icon

### ISSUE THREAD

**Issue by luigino** [kanishka-linux/kawaii-player#7]
Just a minor issue: kawaii-player/tray.png is used as desktop icon, but being just 48x48 it comes out either tiny or grainy on larger resolutions.

Would it be possible to have a bigger one or better yet a svg icon?

**Comment by kanishka-linux** [comment]
yes, **tray.png** might look tiny on larger resolutions. I'll try to provide better icon in future versions. For time being, you can replace **'/usr/share/kawaii-player/tray.png'** with any icon of your choice manually, but keep the name **tray.png**. Changing icon manually won't affect the application.
